docs/azure_arc_jumpstart/azure_arc_sqlsrv/azure/azure_arm_template_sqlsrv_defender/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+type: docs
+title: "Arc-enabled SQL servers - Microsoft Defender for SQL servers on machines"
+linkTitle: "SQL servers on machines - Defender for Cloud"
+weight: 1
+---
+
+## Deploy an Azure Virtual Machine with Windows Server with Hyper-V to support nested virtualization
+
+The following Jumpstart scenario will walk you through on how to use the provided [Azure ARM Template](https://docs.microsoft.com/azure/azure-resource-manager/templates/overview) to deploy an Azure VM installed with Windows Server, setup Hyper-V to support nested virtualization, and create guest VM with SQL Server 2019 on Hyper-V to demonstrate Defender for Cloud for SQL servers on machines and generate alerts for SQL attacks.
+
+By the end of the guide, you will have an Azure VM **ArcBox-Client** installed with Windows Server 2019 with Hyper-V and nested Windows Server VM **ArcBox-SQL** pre-configured with SQL Server 2019, projected as an Azure Arc-enabled SQL Server, then enabled SQL assessment and Microsoft Defender for SQL servers on machines.
```

Comment:
Why is the VM named ArcBox-Client if this is not ArcBox?

Reply:
Same comment for Arcbox-SQL. We should avoid using ArcBox as a name unless this is an actual ArcBox flavor

Reply:
Using a nested VM in Hyper-V for this scenario and used ArcBox as it has built-in capabilities. Any suggestions for the name? I thought of changing name needed after review.

Reply:
Fixed all naming conventions